contracts/GenArt721CoreV3.sol

```diff
@@ -196,6 +198,18 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
         return thisTokenId;
     }
 
+    /**
+     * @notice Internal function that returns whether msg.sender is allowed to
+     * call function with selector `_selector`, as determined by the Admin ACL
+     * contract.
+     * @param _selector Function selector to check.
+     * @dev assumes the Admin ACL contract is the owner of this contract
+     * @dev assumes the Admin ACL contract implements the IAdminACLV0 interface
+     */
+    function _adminAllowed(bytes4 _selector) internal returns (bool) {
+        return IAdminACLV0(owner()).allowed(msg.sender, _selector);
```

Comment:
Storing an explicit `IAdminACLV0` instance on the contract would also allow us to avoid casting here.

Reply:
yes, updated 🎉

Reply:
LGTM! Thank you :)